### ISSUE TITLE
[enhancement]: Make it easy to download 

### ISSUE THREAD

**Issue by magnusviri** [invoke-ai/InvokeAI#6163]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Contact Details

discord: magnusviri

### What should this feature add?

I'd like to be able to download the processed controlnet image (e.g. the black and white canny image) so that I can modify it in something like Photoshop then use the modified image.

**Comment by hipsterusername** [comment]
You can! Hit the save icon on the right of the image.

**Comment by magnusviri** [comment]
I just noticed the tooltip said "Save Control Image". However, clicking the icon does nothing. Maybe this is a bug report instead of a feature request. I'm using the Brave Browser on macOS 13.

**Comment by hipsterusername** [comment]
Hmm - try pressing it twice. If it’s not doing anything that’d definitely be a bug. Should be saving a copy to your images

**Comment by magnusviri** [comment]
Ok, it's working. I'm still learning the interface. I think what happened was I clicked "save" but wasn't expecting it to be added to the Images list. Then I generated a bunch of new images and so the canny image was pushed down and wasn't visible. Then when I really wanted to save the canny image, I tried to save it but because I had already saved it, nothing changed and I didn't get any visual feedback and I assumed it didn't work.
